src/python/pants/backend/python/lint/isort/subsystem.py

```diff
@@ -91,4 +91,6 @@ def config_request(self, dirs: Iterable[str]) -> ConfigFilesRequest:
 
 
 def rules():
-    return collect_rules()
+    return [
```

Comment:
Why is this necessary?

Reply:
oops, forgot to roll this back when rolling back the pilot tool.